config/conf.yaml

```
@@ -0,0 +1,22 @@
+batch_size: 16
+distributed: False
+num_epochs: 3
+accum_iter: 10
+base_lr: 1.0
+max_padding: 72
+warmup: 3000
+device: "cpu"
+data_slice: "4%"
+
+file_prefix: "wmt16_model_"
+model_path: "wmt16_model_final.pt"
+run: "tests"
```

```suggestion
run: transformer-${now:%Y-%m-%d-%H-%M}
```

Having a timestamp in the run name makes it much easier to navigate wandb